docs/introduction.rst

```diff
@@ -5,6 +5,16 @@ Flatpak is a framework for distributing desktop applications across various Linu
 distributions. It has been created by developers who have a long history of working on the
 Linux desktop, and is run as an independent open source project.
 
+Terminology
+-----------
+
+- Flatpak: the framework for distributing desktop applications across various Linux distributions.
```

Comment:
The Flatpak README says "Flatpak is a system for building, distributing, and running sandboxed desktop applications on Linux."

Probably should at least replace "the framework" with "a framework", or just quote the README.